src/shell_minimal.js

```diff
@@ -28,9 +27,6 @@ var Module =
   // Otherwise do a good old typeof check.
   typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {};
 #endif
-
-#else
-var Module = {{{ EXPORT_NAME }}};
```

Comment:
It looks like this reduces code size only for users who do not need to define `Module` in .html file at all. (if user does need to define Module in .html, then the original `var Module = {{{ EXPORT_NAME }}};` definition would be shorter.

Although I think that is ok.

It may be simpler to further condense the code to

```js
#elif SUPPORTS_GLOBALTHIS
var Module = globalThis.{{{ EXPORT_NAME }}} || {};
#else
var Module = typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {};
#endif
```